bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/communication/NuvoConnector.java

```diff
@@ -304,6 +305,11 @@ public void handleIncomingMessage(byte[] incomingMessage) {
             return;
         }
 
+        if (message.contains(PING)) {
```

Comment:
change to:
`if (PING.equals(message))`

Reply:
Sometimes the ping message is preceded by some random characters.  Which is why I did contains.

Reply:
I could make it 'endsWith' which would cover that case and also not capture a "#PING" in some other message somehow..

Reply:
I had not seen that in the experiments I have done with trapping and responding to pings for NuvoNet source communication. Just keep it as contains() just in case.